src/LayerPlan.cpp

```diff
@@ -1010,6 +1010,21 @@ void LayerPlan::addWall(const std::vector<ExtrusionJunction>& wall, int start_id
     }
 }
 
+void LayerPlan::addWall(const std::vector<ExtrusionJunction>& wall, const GCodePathConfig& path_config)
+{
+    ExtrusionJunction junction{*wall.begin()};
```

Comment:
Would segfault if the wall is an empty vector.

Reply:
In theory all empty walls, and therefore empty vectors, are all ready delete and filtered out, before it reaches this point.
What is your opinion about adding an assert, it will make it easier to debug, but won't slow down a release.

Reply:
An assertion is fine, yeah. As long as it has a good error message alongside it.